src/docs/product/accounts/membership.mdx

```diff
@@ -9,87 +9,116 @@ description: "Learn about the different types of Sentry membership, roles, and a
+| **Action**                                                                      | **Billing** | **Member** | **Admin** | **Manager** | **Owner** |
+| ------------------------------------------------------------------------------- | ----------- | ---------- | --------- | ----------- | --------- |
+| Can see and edit billing information and subscription details                   | ✓           |            |           |             | ✓         |
+| Can see and edit legal and compliance details                                   | ✓           |            |           |             | ✓         |
+| Can view and act on issues (such as assigning and resolving)                    |             | ✓          | ✓         | ✓           | ✓         |
+| Can join and leave existing teams                                               |             | ✓          | ✓         | ✓           | ✓         |
+| Can add and remove repositories                                                 |             | ✓          | ✓         | ✓           | ✓         |
+| Can create teams and remove teams from the organization                         |             |            | ✓         | ✓           | ✓         |
+| Can change project settings                                                     |             |            | ✓         | ✓           | ✓         |
+| Can create and remove projects for the organization                             |             |            | ✓         | ✓           | ✓         |
+| Can edit Global Integrations                                                    |             |            | ✓         | ✓           | ✓         |
+| Can add, remove, and change Org Members                                         |             |            |           | ✓           | ✓         |
+| Can change organization settings, including [open membership](#open-membership) |             |            |           | ✓           | ✓         |
+| Can assign Team Admin role                                                      |             |            |           | ✓           | ✓         |
+| Can add, remove, and edit Team Contributors                                     |             |            |           | ✓           | ✓         |
+| Can add projects for teams                                                      |             |            |           | ✓           | ✓         |
+| Can remove projects from teams                                                  |             |            |           | ✓           | ✓         |
+| Can remove an organization                                                      |             |            |           |             | ✓         |
 
-### Member
+## Team-level Roles
 
-- Can view and act on events, as well as view most other data within the organization.
+All members of a specific team and their team-level roles are listed in **Settings > Teams > [Team] > Members**.
```

Comment:
This is listed in both Settings->Members->Member and Settings->Teams->Team->Members